pkg/utils/controller/metrics/metrics.go

```diff
@@ -50,10 +50,17 @@ var (
 		Help: "Number of currently used workers per controller",
 	}, []string{"controller"})
 
+	//FleetPlacementComplete is a prometheus metric which keeps track if the placement is complete
+	FleetPlacementComplete = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
```suggestion
	FleetPlacementCompleteLastTimeStampSeconds = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```
to improve the readability